crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -165,10 +331,193 @@ impl ApiServer {
         Ok(self.mining_engine.set_time(Duration::from_secs(time)))
     }
 
-    fn set_logging(&self, enabled: bool) -> Result<()> {
-        node_info!("anvil_setLoggingEnabled");
+    // Revive RPCs
+    fn eth_chain_id(&self) -> Result<U64> {
+        node_info!("eth_chainId");
+        Ok(U256::from(self.eth_rpc_client.chain_id()).to::<U64>())
+    }
 
-        self.logging_manager.set_enabled(enabled);
-        Ok(())
+    fn network_id(&self) -> Result<u64> {
+        node_info!("eth_networkId");
+        Ok(self.eth_rpc_client.chain_id())
+    }
+
+    fn net_listening(&self) -> Result<bool> {
+        node_info!("net_listening");
+        Ok(true)
+    }
+
+    fn syncing(&self) -> Result<bool> {
+        node_info!("eth_syncing");
+        Ok(false)
+    }
+
+    async fn transaction_receipt(&self, tx_hash: B256) -> Result<Option<ReceiptInfo>> {
+        node_info!("eth_getTransactionReceipt");
+        // TODO: do we really need to return Ok(None) if the transaction is still in the pool?
```

Comment:
Is this what anvil does? I have not looked over eth_rpc_client receipt implementation, and whether it returns Ok(None) if the tx is still in the pool, but if that's what happens and is aligned with anvil, don't see it as a concern we should have.